Gtk-4.0.gir

```diff
@@ -70115,7 +70115,7 @@ and the dark chrome creates too much contrast (web browser, text editor...).</do
         <type name="gboolean" c:type="gboolean"/>
       </property>
       <property name="gtk-cursor-aspect-ratio" writable="1" transfer-ownership="none">
-        <type name="gfloat" c:type="gfloat"/>
```

Comment:
This was changed after 4.0.0? API change...

Reply:
Intentionally it seems: https://gitlab.gnome.org/GNOME/gtk/-/commit/02f06813447cd3a4dfe3472e942bc94a21e5f6c3

Not our problem!